### ISSUE TITLE
Would you like to have mkswap join the org?

### ISSUE THREAD

**Issue by grahamc** [rust-disk-partition-management/gptman#102]
Hey folks,

Great libraries you have here! I have a crate for creating swap filesystems, and I wonder if it would be at home in this org?

Here's a link: https://github.com/grahamc/mkswap/

Thanks for the great work!

**Comment by grahamc** [comment]
I'd be happy to co-own. Also, no sweat if this isn't the right home for mkswap. It is a bit different, for sure.

**Comment by bgilbert** [comment]
Hey, apologies for the delayed response.  I don't have a strong opinion here.  It would indeed expand the scope of the GH org, but I'm not sure how important that is.  The crate is small and narrowly scoped, so the maintenance load should be low.  @cecton, I'm okay with whatever you decide.

**Comment by cecton** [comment]
Yup okay let's do it then. No problem for the delay we're all doing that outside our work right

**Comment by cecton** [comment]
@grahamc @Hoverbear I think you need to initiate the transfer on your side. I will add you to the organization when it's done

**Comment by bgilbert** [comment]
You need to be a member of the organization to perform the transfer.  I'll send you an invite.

**Comment by cecton** [comment]
Sorry I missed the notification

**Comment by cecton** [comment]
Hey @grahamc :wave: what's the status on this? I can see that all the invitations have failed. (Do you have second thought? It's okay I really don't mind if you changed your mind!)
